pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: amplitude_flutter
 description: Amplitude Flutter plugin
-version: 3.8.1
```

Comment:
We actually don't need to bump up the version, since it's more for the dev environment.

Reply:
Yeah, that makes sense.
Solved > https://github.com/amplitude/Amplitude-Flutter/pull/96/commits/29583a742c02e1f959246de31caaf35b12f4a3e3